ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/nova.py

```diff
@@ -34,11 +35,12 @@ def get_productKey(version):
         ('2014.2', version),   # Juno
         ('2015.1', version),   # Kilo
         ('2015.2', version),   # Liberty
```

Comment:
nope, that's intentional- i didn't have a liberty system handy to see which way it returns its version, so i included both possibilities.